### ISSUE TITLE
Map Component should add directions to SO comments automatically

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#721]
_Imported from trac ticket 721.
Created by **rneville**
Opened in trac: Feb 21, 2011, 9:23:28 AM
Last modified in trac: Jul 3, 2014, 2:57:02 PM_
***
_by rneville:_
This is waiting on a deal with a mapping provider

**Comment by kentbower** [comment]
_**kbower** trac status change on Jul 3, 2014, 2:57:01 PM:_ **closed**

**Comment by kentbower** [comment]
_**kbower** trac comment on Jul 3, 2014, 2:57:02 PM:_

Closed as duplicate of [#2778](2778).
